ws/src/hri/CMakeLists.txt

```diff
@@ -0,0 +1,206 @@
+cmake_minimum_required(VERSION 3.0.2)
+project(hri)
+
+## Compile as C++11, supported in ROS Kinetic and newer
```

Comment:
Erase the comments for it to be more readable and make the onboarding easier for new people reading the code. Maybe add a comment like:
```bash
## For further reference on `CmakeLists` check the documentation: https://wiki.ros.org/catkin/CMakeLists.txt
```